src/bundle/Resources/public/scss/_inputs.scss

```diff
@@ -113,8 +113,6 @@
         appearance: none;
         position: relative;
         display: inline-block;
-        vertical-align: middle;
```

Comment:
Note: deleted because middle alignment does not work with multiline labels (checkbox should be aligned with the top line):
| before | after |
|----|---|
| ![Screenshot 2021-12-29 at 10 35 18](https://user-images.githubusercontent.com/10233057/147648014-b5ad8e45-a852-4ff0-898a-82d9bf299eb2.png) | ![Screenshot 2021-12-29 at 10 38 27](https://user-images.githubusercontent.com/10233057/147648251-8e39c9ee-78c6-45df-814a-7c0e53508be8.png) |